frontend/src/app.css

```diff
@@ -0,0 +1,13 @@
+  /* Image Widget */
```

Comment:
I'm wondering if our intent would be better and provide a better pattern to add this file to `src/styles`?